annotation-protocol/tools/protocol-server.py

```diff
@@ -312,7 +312,7 @@ def annotation_head(request, response):
         response.status = 404
 
     add_cors_headers(response)
-    response.content = "Annotation Options\n"
+    # response.content = "Annotation Options\n"
```

Comment:
I'd rather this was just gone rather than commented out, unless there's any expectation it'll ever come back (in which case there should be a comment saying when it will!).

(That said, if you don't change that before I go to bed I'll merge anyway. :))